securedrop/source.py

```diff
@@ -263,7 +262,7 @@ def submit():
         if entropy_avail >= 2400:
             async_genkey(g.sid, g.codename)
 
-    g.source.last_updated = datetime.utcnow()
+    g.source.last_updated = datetime.now().date()
```

Comment:
All of the dates that get stored should be normalized to UTC. Any reason you can't use `utcnow().date()`?